### PR TITLE
rename asset file

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -64,15 +64,13 @@ jobs:
...
-        if: ${{ inputs.tag_name == '' && github.ref_name == 'main' }}
         uses: actions/upload-artifact@v3
         with:
-          name: thyra-plugin-wallet_${{ matrix.target }}-${{ matrix.arch }}
-          path: ${{ env.TARGET_NAME }}${{ matrix.ext }}
-      - name: Zip binary
-        if: ${{ inputs.tag_name != '' }}
-        run: zip $TARGET_NAME.zip $TARGET_NAME${{ matrix.ext }}
+          name: ${{ env.TARGET_NAME }}.zip
+          path: ${{ env.TARGET_NAME }}.zip
```
Comment (peterjah) [inline .github/workflows/build.yml:73]: There is a probleme here. upload artifact action takes some files and put it into a zip.
So here you are creating a zip into a zip

**Review by peterjah — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (Thykof) [reply .github/workflows/build.yml]: I removed the commit that bring this change so that we can merge it asap

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ jobs:
     # The build job runs on the specified operating system.
     runs-on: ${{ matrix.os }}
     env:
-      TARGET_NAME: thyra-plugin-wallet_${{ matrix.target }}-${{ matrix.arch }}
+      TARGET_NAME: wallet-plugin_${{ matrix.target }}-${{ matrix.arch }}
```
Comment (mazmassa) [inline .github/workflows/build.yml:46]: Is it possible keep the dashed mode in the all sentence!?

```suggestion
      TARGET_NAME: wallet-plugin-${{ matrix.target }}-${{ matrix.arch }}
```

**Review by mazmassa — APPROVED** [review]
LGTM!

File: .github/workflows/build.yml (hunk 2 above)
Comment (mazmassa) [reply .github/workflows/build.yml:46]: Forget... notice that is the file name in the end!

**Pull request merged**